packages/dev/core/src/Buffers/buffer.ts

```diff
@@ -1039,7 +1039,8 @@ export class VertexBuffer {
                 const remainder = offset % 4;
 
                 if (remainder) {
-                    offset = Math.max(0, offset - remainder);
+                    // If not aligned, copy the data to aligned buffer

```

Comment:
I'm okay with this fix, but it might be beneficial to log a warning as this is a performance hit. We also need to duplicate this fix [here](https://github.com/BabylonJS/Babylon.js/blob/27b194fb39a3dca21d8610440c73b644e3c3d1de/packages/dev/core/src/Buffers/bufferUtils.ts#L46-L50).